src/main/java/ca/nrc/cadc/beacon/web/resources/FolderItemServerResource.java

```diff
@@ -178,13 +186,16 @@ private void move(final VOSURI source, final VOSURI destination) throws IOExcept
                              final ClientTransfer clientTransfer =
                                  voSpaceClient.createTransfer(
                                      transfer);
-                             clientTransfer.setMonitor(false);
+                             clientTransfer.setMonitor(true);
```

Comment:
This will time out for large moves.  Is this intended?

Reply:
No. Trick is the move errors are only reported through the uws job errorSummary. Which is only available after a job finishes (regardless of the state.) - How large a move?